LICENSE.md

```diff
@@ -0,0 +1,21 @@
+# MIT License
+
+Copyright (c) 2019 Miguel Morin
```

Comment:
Is this correct? I would have thought that the copyright holder is either the Institute or all the authors.

Reply:
You are right, it's incorrect. I tested `usethis::use_mit_license()` with my name and changed it to the Institute's only in `LICENSE`, not in `LICENSE.md`.